### ISSUE TITLE
BigQuery procedures using 'IF' statement fail

### ISSUE THREAD

**Issue by ferrisnorth** [flyway/flyway#3261]
It appears as though BigQuery procedures using 'IF' statements fail.

  Current JRE version ...................... openjdk-14_windows-x64
  Current Flyway version ................... 7.14.0
  Platform ................................. GCP
  Database ................................. BigQuery
  Required Drivers ......................... GoogleBigQueryJDBC42.jar
  Other JDBC Stuff ......................... SimbaJDBCDriverforGoogleBigQuery42_1.2.18.1022.zip
  dbConnector Version ...................... v2:443
  JDBC Timeout ............................. 600
  
**SQL Migration Text :**
"
  CREATE OR REPLACE PROCEDURE tst.tst_proc_if()
    BEGIN
        IF 1 = 1 THEN
            SELECT 'ok';
        ELSE
            SELECT 'bah';
        END IF;
    END;
"

**ERROR:** 
" ... 
Caused by: Migration R__01_00_tst_proc_if.sql failed
-----------------------------------------
SQL State  : HY000
Error Code : 100032
Message    : [Simba][BigQueryJDBCDriver](100032) Error executing query job. Message: Syntax error: Expected ";" but got end of script at [7:15]
Location   : .\repeatable\R__01_00_tst_proc_if.sql (\repeatable\R__01_00_tst_proc_if.sql)
Line       : 1
Statement  : CREATE OR REPLACE PROCEDURE tst.tst_proc_while()
    BEGIN
        IF 1 = 1 THEN
            SELECT 'ok';
        ELSE
            SELECT 'bah';
        END IF

Caused by: java.sql.SQLException: [Simba][BigQueryJDBCDriver](100032) Error executing query job. Message: Syntax error: Expected ";" but got end of script at [7:15]
Caused by: com.simba.googlebigquery.support.exceptions.GeneralException: [Simba][BigQueryJDBCDriver](100032) Error executing query job. Message: Syntax error: Expected ";" but got end of script at [7:15]
Caused by: com.google.api.client.googleapis.json.GoogleJsonResponseException: 400 Bad Request
POST https://bigquery.googleapis.com/bigquery/v2/projects/***/queries
{
  "code" : 400,
  "errors" : [ {
    "domain" : "global",
    "location" : "q",
    "locationType" : "parameter",
    "message" : "Syntax error: Expected \";\" but got end of script at [7:15]",
    "reason" : "invalidQuery"
  } ],
  "message" : "Syntax error: Expected \";\" but got end of script at [7:15]",
  "status" : "INVALID_ARGUMENT"
}"

**Comment by ferrisnorth** [comment]
I have a few client procedures that are failing, and so I started testing for what is breaking. It appears as though it would be good for someone to at least go through the documented supported BQ statements here <https://cloud.google.com/bigquery/docs/reference/standard-sql/scripting> 

I tested a couple and found parsing errors. BigQuery migrations are so very very slow though that I don't really have time to test all of these use cases. Maybe the QA team can run tests faster with the "teams" version.
